highspy/meson.build

```diff
@@ -2,7 +2,8 @@ py_mod = import('python')
 py = py_mod.find_installation(pure: false)
 pyb11_dep = [
   py.dependency(),
-  dependency('pybind11')
+  dependency('pybind11',
+            required: true)
```

Comment:
Note that `required: true` is the default, you can omit it although some people would say "explicit is better than implicit". Just pointing out that this commit didn't *change* that, only made it... explicit.